sig-docs/charter.md

```diff
@@ -0,0 +1,76 @@
+# SIG Docs Charter
+
+This charter adheres to the conventions described in the [Kubernetes Charter README] and adheres to the Roles and Organization Management specified in [sig-governance].
+
+## Scope
+
+SIG Docs is responsible for Kubernetes documentation.
+
+The responsibility for feature documentation lies with feature developers. SIG Docs sets standards for feature documentation, provides clear paths for docs contribution, and coordinates documentation updates during quarterly releases.
+
+We maintain the Kubernetes website's infrastructure and tooling.
+
+We create subprojects as needed to handle specific aspects of Kubernetes documentation.
+
+### In scope
+
+SIG [readme]
+
+#### Code, Binaries and Services
+
+- The kubernetes.io website:
+
+  - Infrastructure and tooling
+  - Internationalization and localization support
+  - Documentation and release note training for other sigs
+  - Generated reference documentation and review of related strings in code for:
+    - The core Kubernetes APIs
+    - Kubernetes command-line tools, including but not limited to kubectl
+    - Kubernetes setup tools
+    - The Federation API
+
+See the next section for limitations on how the SIG helps with related efforts on other projects.
+
+#### Cross-cutting and Externally Facing Processes
+
+- [Standards for content][standards]: style guide, contributor guide, PR reviews
+  - Note that the SIG is not responsible for branding any content related to the Kubernetes organization. We are sometimes informally invited to consult on branding and design decisions, but we have no formal responsibility for such work.
+- [Coordinating docs contributions for quarterly releases][docs-release]
+- Blog PR management
+- Reorganization and rewrite of technical content to improve UX and technical accuracy based on current and recent releases, based on:
+    - user-reported issues
+    - site analytics
+    - content audits performed by SIG members
+
+The SIG is sometimes asked to advise on Kubernetes components that are not part of the Kubernetes/Kubernetes GitHub repository, or on other Kubernetes subprojects, but we are not responsible for the documentation of any of these components or projects.
+
+### Out of scope
+
+- Draft documentation of new features -- that is, new technical content, especially for features we are not familiar with. Specifically, during each release cycle individual feature SIGs are responsible for submitting documentation PRs for the features they contribute to the release. It is not the responsibility of SIG-docs to provide this content. We do, however, offer advice about information architecture for new features in the docs, work closely with some SIGs on individual docs PRs, and otherwise provide oversight to make sure that new feature documentation meets the standards of SIG docs. 
```

Comment:
My understanding is that SIG docs doesn't write the documentation, but is responsible for reviewing it and coordinating it.  e.g. do docs prs require a reviewer from sig-docs to approve?  I would strengthen the responsibility from offering advise to `review and coordinate accepting the documentation written by community members.`